vllm/entrypoints/openai/protocol.py

```diff
@@ -103,7 +103,8 @@ class ResponseFormat(OpenAIBaseModel):
 
 
 class StreamOptions(OpenAIBaseModel):
-    include_usage: Optional[bool]
+    include_usage: Optional[bool] = False
+    continuous_usage_stats: Optional[bool] = False
```

Comment:
@simon-mo Are we happy to keep the default behaviour as-is? Or would we like to enable continuous usage stats by default?

Reply:
I think adding few more bytes to the streaming message should not harm the inter-token latency, so enabling it as a default sounds good to me.

Reply:
done